public/robots.txt

```diff
@@ -0,0 +1,9 @@
+# *
```

Comment:
Do we need `robots.txt` in the public folder? I think a sitemap plugin should generate this after the build 🤔 
I don't want to have robots e.g. on staging (or it should have different content).